<?xml version="1.0" encoding="UTF-8"?>
<interface xmlns="http://xml.phpdox.net/src" full="Symfony\Component\Yaml\Exception\ExceptionInterface" namespace="Symfony\Component\Yaml\Exception" name="ExceptionInterface" abstract="false" final="false" start="19" end="21">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/yaml/Exception" file="ExceptionInterface.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/yaml/Exception/ExceptionInterface.php" size="443" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="5c201b9a693fecc2114c80d74737b3450e39eada" relative="vendor/symfony/yaml/Exception/ExceptionInterface.php"/>
  <docblock>
    <description compact="Exception interface for all exceptions thrown by the component."/>
    <author value="Fabien Potencier &lt;fabien@symfony.com&gt;"/>
  </docblock>
</interface>
